README.md

```diff
@@ -72,24 +72,8 @@ This curated list is on very early stage. So, let's make it together!
    - Adds a button to the Toolbar with a dropdown that allows case converting in any text in the editor.
  - [summernote-cleaner](https://github.com/DiemenDesign/summernote-cleaner)
    - Adds a Button to toolbar that when used cleans the MS Word Crud from the editor text.
- - [summernote-image-attributes](https://github.com/DiemenDesign/summernote-image-attributes)
-   - This is a forked and extended version of "summernote-image-title", for those that want a little more control over editable attributes for Images.
- - [summernote-image-captionit](https://github.com/DiemenDesign/summernote-image-captionit)
-   - Allows adding figure and figcaption to images, with removal when selected twice.
- - [summernote-image-shapes](https://github.com/DiemenDesign/summernote-image-shapes)
-   - Adds option with dropdown to the Image Popover to add or remove Bootstrap Image Classes for styling the look and behaviour of images.
- - [summernote-image-title](https://github.com/asiffermann/summernote-image-title)
-   - Adds a button to the image popover to edit title and alt attributes.
- - [summernote-list-styles](https://github.com/tylerecouture/summernote-list-styles)
-   - Adds a toolbar dropdown menu to change the number/bullet style of HTML lists.
```

Comment:
It looks like you deleted other repositories, not yours. Was it intended or not?

Reply:
No that wasn't intended.

Reply:
@DiemenDesign So, won't you revert that? :)

Reply:
I got busy and forgot about it.